exposee-app/server.js

```diff
@@ -65,7 +113,21 @@ app.post('/videos', async (req, res) => {
   res.status(500).json({ message: error.message });
   }
 });
-
+app.post('/broadcast', async (req, res)=> {
+  const userId = req.body.id;
+  try{
+    const user = await User.findByPk(userId);
+    if(!user){
+      return res.status(404).json({message: 'User not found'});
+    }
+    user.isLive = true;
```

Comment:
do we have an isLive on the user? 

When a user clicks broadcast, we don't want to turn the user on to is live
we want to create a video record,

so I think on the next iteration of this, let's change it to actually create that video, which you'll need for your technical challenge!